### DISCUSSION TITLE
Adjacency Matrix

### DISCUSSION THREAD

**Discussion by KERob20** [GreenleafLab/ArchR#1008] [Questions / Documentation]
Hi ArchR Team, 

I've been looking through the documentation for the package and the dependencies, and I was wondering if there is any function that will call/generate an adjacency/connectivity matrix from the UMAP? 

Thanks for you help!

**Comment by rcorces** [comment]
I'm not sure I understand. I'm only familiar with adjacency matrices from graph theory and I'm not sure how this applies to the reduced dimension space or UMAP which does not have edges. Are you just looking for nearest neighbors?

**Comment by KERob20** [comment]
My apologies; I'm still relatively new to this field. 
Yes, I'm after the nearest neighbours. 

Thanks

**Reply by rcorces** [reply]
ArchR does not provide a function to do this but you can try the `nabor::knn()` function
